CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Minor: Stream status requests are now batched. (#4713)
 - Minor: Added `/c2-theme-autoreload` command to automatically reload a custom theme. This is useful for when you're developing your own theme. (#4718)
 - Minor: Show channel point redemptions without messages in usercard. (#4557)
+- Minor: Allow for customizing the behavior of `Right Click`ing of usernames. (#4622, #4751)
```

Comment:
note: this changelog was accidentally dropped in 2.4.4 but was not apart of said version, and since this PR is a followup it seems correct to move it back in this PR